ow_plexil/rqt_planner/scripts/rqt_plexil_planner.py

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env python
+
```

Comment:
Please add license text here. After the shebang is fine, since those have to be the very first line.